Use ".Bx DragonFly" to represent DragonFlyBSD since Debian groff does
not have the ".Dx" macro yet.

--- a/dma.8
+++ b/dma.8
@@ -269,7 +269,7 @@
 The
 .Nm
 utility first appeared in
-.Dx 1.11 .
+.Bx DragonFly 1.11 .
 .Sh AUTHORS
 .Nm
 was written by
